djangocms_pageadmin/admin.py

```diff
@@ -394,18 +395,21 @@ def set_home_view(self, request, object_id):
 
     def get_urls(self):
         info = self.model._meta.app_label, self.model._meta.model_name
-        return [
+        # we replace the duplicate with our function.
+        old_urls = [v for v in super().get_urls() if 'duplicate' not in str(v.name)]
+        new_urls = [
             url(
                 r"^(.+)/duplicate-content/$",
                 self.admin_site.admin_view(self.duplicate_view),
```

Comment:
I will create a new PR in djangocms-versioning that picks up this change so that pageadmin is not required.